AstroAdapt.GraphQL/Query.cs

```diff
@@ -13,24 +12,24 @@ public class Query
         /// <summary>
         /// Gets all available inventory.
         /// </summary>
-        /// <param name="factory">The DbContext factory.</param>
+        /// <param name="context">The DbContext.</param>
         /// <returns>The list of inventory.</returns>
         [UseProjection]
         [UseFiltering]
         [UseSorting]
-        public IQueryable<Component> GetInventory([Service] IDbContextFactory<AstroContext> factory)
-            => factory.CreateDbContext().Components;
+        public IQueryable<Component> GetInventory(AstroContext context)
+            => context.Components;
 
         /// <summary>
         /// Gets all manufacturers.
         /// </summary>
-        /// <param name="factory">The DbContext factory.</param>
+        /// <param name="context">The DbContext.</param>
         /// <returns>The list of manufacturers.</returns>
         [UseProjection]
         [UseFiltering]
         [UseSorting]
-        public IQueryable<Manufacturer> GetManufacturers([Service] IDbContextFactory<AstroContext> factory)
-            => factory.CreateDbContext().Manufacturers;
+        public IQueryable<Manufacturer> GetManufacturers(AstroContext context)
```

Comment:
Since we registered the `AstroContext` as pooled DBContext in our GraphQL config we can just inject the `AstroContext` here and the execution engine will take care of renting the DBContext and returning it.